articles/api-auth/tutorials/use-one-rs-for-apis.md

```diff
@@ -0,0 +1,108 @@
+---
+
+# Represent Multiple APIs as a Single Resource Server
+
+If you have multiple APIs, you can treat them as a single Resource Server. Doing this allows you to implement just one authentication flow. You can then control access to the individual APIs by assigning the appropriate scopes.
+
+This article shows you how to use and represent multiple APIs as a single Resource Server in Auth0 using a [sample application you can download](https://github.com/auth0-samples/auth0-api-auth-implicit-sample) if you would like to follow along as you read.
+
+## The Sample Application
+
+In the sample application, we have:
+
+* 2 Node.js APIs called `contacts` and `calendars`;
+* 1 resource server representing the 2 APIs mentioned above. This is the Auth0 API we will call `Organizer Service`;
+* Namespaced scopes to access the APIs. We will work with two (`read:contacts` and `read:calendar`), but you can create any number of scopes when you implement your specific business scenario;
+* The [Implicit Grant flow](/api-auth/grant/implicit), which we use to obtain an `access_token` that works with both APIs.
+
+Please see the `README` for additional information on setting up the sample on your local environment.
+
+## Create the Auth0 API
+
+Log in to your Auth0 Dashboard, and navigate to the APIs section.
+
+::: panel-info Auth0 APIs
+Please see [APIs](/apis) for detailed information on working with APIs in the Dashboard.
+:::
+
+Click **Create API**.
+
+![](/media/articles/api-auth/tutorials/multiple-apis-one-resource-server/dashboard-apis.png)
+
+You will be prompted to provide a **name** and **identifier**, as well as choose the **signing algorithm**, for your new API.
+
+For the purposes of this article, we'll call our API `Organizer Service` and set its unique identifier to `organize`. By default, the signing algorithm for the tokens this API issues is **RS256**, which we will leave as is.
+
+![](/media/articles/api-auth/tutorials/multiple-apis-one-resource-server/create-new-api.png)
+
+Once you've provided the required details, click **Create** to proceed.
+
+### Configure the Auth0 API
+
+After Auth0 creates your API, you'll be directed to its *Quick Start* page. At this point, you'll need to create the appropriate **Scopes**, which you can do via the *Scopes* page.
+
+![](/media/articles/api-auth/tutorials/multiple-apis-one-resource-server/scopes-page.png)
+
+Scopes allow you to define the API data accessible to your client applications. You'll need one scope for each API represented and action. For example, if you want to `read` and `delete` from an API called `samples`, you'll need to create the following scopes:
+
+* `read:samples`
+* `delete:samples`
+
+For our sample application, we'll add two scopes:
+
+* `read:calendar`;
+* `read:contacts`.
+
+You can think of each one as a microservice.
+
+![](/media/articles/api-auth/tutorials/multiple-apis-one-resource-server/new-scopes.png)
+
+At this point, you've:
```

Comment:
I would remove this, it looks redundant. Just add something like "add these two scopes and save your changes"